src/components/Header/Chat.tsx

```diff
@@ -0,0 +1,194 @@
+      >
+        {showChat && (
+          <div
+            id="chat_container"
+            className=" overflow-auto mx-auto  flex flex-col h-full "
+          >
+            <div className="flex flex-col space-y-2 pb-44 p-5 flex-grow">
+              {chatLog.length === 0 ? (
+                <div className="text-gray-500">
+                  Welcome to Balancer! <br />
+                  <br />
+                  Balancer is a powerful open-source tool available for free
+                  use. Simply start typing your questions or concerns, and we'll
+                  do our best to assist you. <br />
+                  <br />
+                  {/* <br />
+                  <br /> Balancer is an assistive tool and cannot be used as a replacement for a real human prescriber. */}
+                </div>
+              ) : (
+                chatLog.map((message, index) => (
+                  <div
+                    key={index}
+                    className={`flex ${
+                      message.type === "user" ? "justify-end" : "justify-start"
+                    }`}
+                  >
+                    <div
+                      className={`${
+                        message.type === "user"
+                          ? "bg-blue-200 text-black "
+                          : "text-black bg-gray-200 border-2 "
+                      }rounded-lg p-2 max-h-[100%] max-w-[500px]`}
+                    >
+                      {message.message}
+                    </div>
+                  </div>
+                ))
+              )}
+              {isLoading && (
+                <div key={chatLog.length} className="flex justify-between">
+                  <div className="rounded-lg p-4 text-white max-w-sm">
+                    <TypingAnimation />
+                  </div>
+                </div>
+              )}
+            </div>
+
+            <div className="absolute bottom-0 left-0 right-0 p-4 rounded-lg border-t bg-white">
+              <div className=" space-x-2  p-2 flex ">
+                {suggestionPrompts.map((suggestion, index) => (
+                  <button
+                    type="button"
+                    key={index}
+                    className="rounded-md border  hover:bg-blue-200 p-2"
+                    onClick={() => setInputValue(suggestion)}
+                  >
+                    {suggestion}
+                  </button>
+                ))}
+              </div>
+              <form onSubmit={handleSubmit} className="flex">
+                <div className="flex-grow ml-2">
+                  <input
+                    type="text"
+                    className="pl-3 py-2 px-2 w-full font-semibold placeholder-gray-500 text-black  bg-white ring-1 ring-slate-1000/10 hover:ring-slate-300 focus:outline-none focus:ring-2 focus:ring-sky-500 shadow-sm rounded-lg  text-slate-400 dark:bg-slate-800 dark:ring-0 dark:text-slate-300 dark:highlight-white/5 dark:hover:bg-slate-700"
```

Comment:
when there's this many tailwind classes, it might be good to hide behind a custom class, so we're not muddying up the markup as much.